modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/api/construction/ConstructionMetadata.scala

```diff
@@ -0,0 +1,18 @@
+package org.tessellation.rosetta.domain.api.construction
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.rosetta.domain.amount.Amount
+import org.tessellation.schema.transaction.TransactionReference
+
+import derevo.cats.show
+import derevo.circe.magnolia.customizableDecoder
+import derevo.derive
+
+object ConstructionMetadata {
+
+  @derive(customizableDecoder, show)
+  case class TransactionMetadata(
+    lastReference: TransactionReference,
+    suggestedFee: Option[Amount]
+  )
```

Comment:
do we have a `/construction/metadata` endpoint already implemented?
looks like this schema `lastReference + suggestedFee` is different than the one from the rosetta specification: https://www.rosetta-api.org/docs/models/ConstructionMetadataResponse.html